website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -314,6 +314,8 @@ make docker-build docker-push IMG=quay.io/$USERNAME/memcached-operator:v0.0.1
 **Note**: The name and tag of the image (`IMG=<some-registry>/<project-name>:tag`) in both the commands can also be set in the Makefile.
 Modify the line which has `IMG ?= controller:latest` to set your desired default image name.
 
+**Note**: You need [kube-builder][kube-builder] installed as the `docker-build` target uses it for the controller test suite.
```

Comment:
See the comment https://github.com/operator-framework/operator-sdk/pull/4395/files#r560166486. IMO it should not be added at all.

Reply:
Updated as per https://github.com/operator-framework/operator-sdk/pull/4395#discussion_r561128596.